src/lib/api/api.go

```diff
@@ -438,31 +437,38 @@ func (a *API) ModifyIntervalDuration(w http.ResponseWriter, r *http.Request) {
 	}
 	var req struct {
 		IntervalName string `json:"intervalName"`
-		Value        string `json:"value"`
+		Value        uint32 `json:"value"`
 	}
 	decoder := json.NewDecoder(r.Body)
 	if err := decoder.Decode(&req); err != nil {
-		panic(err)
+		a.ResponseStatus = "400"
+		a.Data = "Could not modify the config file. Value not a uint or intervalName is not a string"
+		w.WriteHeader(http.StatusBadRequest)
+		a.send(w, a.marshalled())
```

Comment:
You will need to update from master branch and then push the recent commits. There have been changes related to send() and a.Data without with, there would be compilation errors.

Reply:
@Harkishen-Singh Do I need to make new PR?

Reply:
No, a git rebase from upstream master should be enough and then you may have to resolve the conflicts.

Reply:
ok